apps/avifenc.c

```diff
@@ -2480,17 +2474,11 @@ int main(int argc, char * argv[])
                 lossless = AVIF_FALSE;
             }
 
-            avifBool matrixCoefficientsAreLosslessCompatible = usingIdentityMatrix;
-#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
-            matrixCoefficientsAreLosslessCompatible |= (image->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_YCGCO_RE ||
-                                                        image->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_YCGCO_RO);
-#endif
+            avifBool matrixCoefficientsAreLosslessCompatible = usingIdentityMatrix ||
+                                                               (image->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_YCGCO_RE ||
+                                                                image->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_YCGCO_RO);
```

Comment:
Optional: The variable `matrixCoefficientsAreLosslessCompatible` is no longer necessary.

Reply:
This one makes the condition below clearer IMHO so keeping it.

Reply:
You are right. The variable could be declared as const.